[DOCS-15222] Document Java agentless Feature Flags delivery (#38518)
DataDog/documentation: content/en/feature_flags/concepts/configuration_sources.md

| Environment variable | Default | Description |
|---|---|---|
| `DD_FEATURE_FLAGS_CONFIGURATION_SOURCE_AGENTLESS_BASE_URL` | Datadog-managed endpoint | Overrides the agentless flag configuration endpoint or base URL. See [Use a custom agentless endpoint](#use-a-custom-agentless-endpoint). |

> | `DD_FEATURE_FLAGS_CONFIGURATION_SOURCE_AGENTLESS_POLL_INTERVAL_SECONDS` | `30` | Positive integer that sets the time between completed polling attempts, capped at 3600 seconds in Node.js. |

Author: Applied: documented the Node.js cap and clarified that Java does not apply it in d75bee81d1.

> | `DD_FEATURE_FLAGS_CONFIGURATION_SOURCE_AGENTLESS_POLL_INTERVAL_SECONDS` | `30` | Positive integer that sets the time between completed polling attempts. Node.js caps values at 3600 seconds. Java does not apply this cap. |

Reviewer: ```suggestion
| `DD_FEATURE_FLAGS_CONFIGURATION_SOURCE_AGENTLESS_POLL_INTERVAL_SECONDS` | `30` | Positive integer that sets the time between completed polling attempts. Java does not limit attempts, while Node.js caps values at 3600 seconds. |
```
We have a precedent going by now on this page to mention Java first, then Node.js, so it's easier to read if we just stick with it 🙂
